src/FormModel.php

```diff
@@ -260,6 +265,7 @@ private function addErrors(array $items): void
     private function attributes(): array
     {
         $class = new \ReflectionClass($this);
+        $this->attributes = [];
```

Comment:
Looks like side effect.is it necessary to modify the form in a query method?

Reply:
[This line](https://github.com/yiisoft/form/blob/edfeaf089d54cb9865ad4d7e634dccb170b63d8d/src/FormModel.php#L284) read the property, but if we have no one property it will throw error.